src/test/java/com/wikia/webdriver/common/core/url/UrlBuilderTests.java

```diff
@@ -0,0 +1,56 @@
+package com.wikia.webdriver.common.core.url;
+
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.core.configuration.EnvType;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+
+public class UrlBuilderTests {
+
+    @BeforeTest
+    public void setConfig() {
+        Configuration.setTestValue("env", "sandbox-s1");
+        Configuration.setTestValue("forceHttps", "false");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "false" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)